lucille-core/src/main/java/com/kmwllc/lucille/core/JsonDocument.java

```diff
@@ -511,6 +516,35 @@ public List<byte[]> getBytesList(String name) {
     return result;
   }
 
+  @Override
+  public JsonNode getJson(String name) {
```

Comment:
Need to test how this behaves when we add a JsonNode that happens to be an ArrayNode as a single value on a document and then try to get it back via getJson(String name). Ideally we'd get back the ArrayNode itself -- the same thing we added.

However getSingleNode() is going to check isMultiValued() which is going to check whether we're looking at an ArrayNode. The document will think the ArrayNode represents multiple values and getJson() will return the first contained value in the ArrayNode?  I believe this is why we didn't implement these json getters and setters before. It's not clear how to distinguish an ArrayNode that should be treated as a single value versus an ArrayNode that was created internally by JsonDocument to represent a multi-valued field.